src/Forms/book-form/BookForm.jsx

```diff
@@ -1,15 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Style from './BookForm.module.css';
 
-/* eslint-disable jsx-a11y/anchor-is-valid, react/jsx-curly-brace-presence */
-export default function BookForm() {
+/* eslint-disable jsx-a11y/anchor-is-valid, react/jsx-curly-brace-presence, react/prop-types */
```

Comment:
- [ ] As pointed out above, As required by Microverse you are not meant to use any linter `disable rule` in your codebase, the linters are there to make sure your code follows the best practices. I'll suggest you remove the linters disable rule and fix the errors. :+1: :100: 

## I'll suggest you go through your code and remove all the linters diable rule in every component too. the ones in the `Header, Footer` component and the `index.js` file :+1: :wave: